facturx/facturx.py

```diff
@@ -147,3 +147,111 @@ def write_xml(self, path):
         with open(path, 'wb') as f:
             f.write(self.xml_str)
 
+    def __make_dict(self):
+        dict_data = xml_flavor.FIELDS
+        required_list = []
```

Comment:
Don't build a list beforehand. Check if it's required when you need it.

Reply:
Should I use list comprehension instead?

Reply:
Should I use list comprehension?